sorc/fre-nctools.fd/shared_lib/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+set(c_src
+    create_xgrid.c
+    gradient_c2l.c
+    interp.c
+    mosaic_util.c
+    mpp.c
+    mpp_domain.c
+    mpp_io.c
+    mpp_domain.c
+    read_mosaic.c
+    tool_util.c)
+
+if(CMAKE_C_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_C_FLAGS "-traceback")
+  set(CMAKE_C_FLAGS_RELEASE "-O2")
+  set(CMAKE_C_FLAGS_DEBUG "-O0")
+elseif(CMAKE_C_COMPILER_ID MATCHES "^(GNU|Clang|AppleClang)$")
+  set(CMAKE_C_FLAGS " ")
+  set(CMAKE_C_FLAGS_RELEASE " ")
+  set(CMAKE_C_FLAGS_DEBUG " ")
+endif()
+
+add_library(shared_all STATIC ${c_src})
```

Comment:
This directory is called `shared_lib`. To be consistent and easily relatable, call the library with the same name.

Reply:
Good idea.  Done.